examples/README.md

```diff
@@ -6,6 +6,8 @@ noisy GPS and odometry sensors. In this example the noise is state dependent, an
 learn the cost weight as a function of pose. 
 - **motion_planning_2d.py:** Is an example of how to set up a differentiable motion planning
 problem, inspired by [Bhardwaj et al. 2020](https://arxiv.org/pdf/1907.09591.pdf).
+- **se2_planning.py:** Brief example of how to use SE2 variables with the motion planner
+utility, to handle poses with angles in 2D maps.
```

Comment:
Also add that it handles nonholonomic cost.

Reply:
Fixed in 9a7cb885bfffa9bdb2fd8d41c77bf6563f61e666.